docs/design-tokens/index.html

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
I've written this page for some reason but I haven't exposed it in the nav for now as there aren't any public tokens for anyone to use so it's a bit useless